### DISCUSSION TITLE
Radiator termostat 'TS0601' and manufacturer name '_TZE200_lllliz3p'

### DISCUSSION THREAD

**Discussion by mitjasl** [Koenkk/zigbee2mqtt#10873] [Devices]
Hi,

I got this termostat, but is not recognized by zigbee2mqtt:

warn  2022-01-17 18:49:50: Received message from unsupported device with Zigbee model 'TS0601' and manufacturer name '_TZE200_lllliz3p'
warn  2022-01-17 18:49:50: Please see: https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html

It looks the same as this one: https://www.zigbee2mqtt.io/devices/TS0601_thermostat_1.html

What is the procedure to add it to z2m database?

Thanks a lot.

**Comment by mitjasl** [comment]
Well, I found this commit, device is recognized now :)
https://github.com/Koenkk/zigbee-herdsman-converters/commit/a108f49a7bf0c9d94104789464b82d8d0b819bbd

**Comment by zsoltty91** [comment]
Dear Devs!

I have the same issue, can you help me to get work my thermostat with zigbee2mqtt.
Log:
Received message from unsupported device with Zigbee model 'TS0601' and manufacturer name '_TZE200_bvu2wnxz'

Thanks!

**Reply by stevenbuehner** [reply]
I am having the same issue with the same manufacturer and created a pull request to solve this in Koenkk/zigbee-herdsman-converters#5066
